lib/spack/spack/schema/config.py

```diff
@@ -94,6 +94,14 @@
             "additional_external_search_paths": {"type": "array", "items": {"type": "string"}},
             "binary_index_ttl": {"type": "integer", "minimum": 0},
             "aliases": {"type": "object", "patternProperties": {r"\w[\w-]*": {"type": "string"}}},
+            "includes": {
+                "type": "array",
+                "default": [],
+                "items": {
+                    "type": "object",
+                    "properties": {"when": {"type": "string"}, "file": {"type": "string"}},
```

Comment:
This says `file:` but the PR description says `path:` -- I like `path:` better as we could include either a single config file or a path to a config directory.